designsafe/apps/api/agave/filemanager/public_search_index.py

```diff
@@ -26,7 +26,7 @@
 
 class PublicationIndexed(DocType):
     class Meta:
-        index = settings.ES_INDICES['publications']['name']
+        index = settings.ES_INDICES['publications']['alias'][0]
```

Comment:
Can we avoid hard coding a list index?

Reply:
Would it be OK to tweak `settings.ES_INDICES` to be a string instead of a list? AFAIK we've never had a case where we've actually wanted multiple aliases for one index.

Reply:
Yeah, that's ok.

Reply:
Cool, I pushed up that change.